src/basics/Locale/index.ts

```diff
@@ -0,0 +1,8 @@
+export const createLocalizedString = (strings = {}) => (key, ...rest) => {
```

Comment:
Is this something we want to establish as a pattern?  Currently we don't have plans for localized strings?

Reply:
This pattern will not require you to localize strings, but will keep all strings in file sections, which make them easier to change, and makes easier to switch to localization if we would have such plan in future. 
Most problems with localizations are usually come from struggle to find all strings and get developer used to t("string") notation.
Proper localization will require files to be located in folders per language - but we won't go there until we will start to really discuss localization.